src/transaction-flow/input/RevokePermissions/RevokePermissions-flow.tsx

```diff
@@ -20,6 +21,7 @@ import { dateTimeLocalToDate, dateToDateTimeLocal } from '@app/utils/datetime-lo
 
 import { ControlledNextButton } from './components/ControlledNextButton'
 import { GrantExtendExpiryView } from './views/GrantExtendExpiryView'
+import { NameConfirmationWarningView } from './views/NameConfirmationWarningView'
 import { ParentRevokePermissionsView } from './views/ParentRevokePermissionsView'
 import { RevokeChangeFusesView } from './views/RevokeChangeFusesView'
 import { RevokeChangeFusesWarningView } from './views/RevokeChangeFusesWarningView'
```

Comment:
<img width="1141" alt="Screenshot 2024-04-15 at 1 20 06 PM" src="https://github.com/ensdomains/ens-app-v3/assets/1967258/3617746c-8683-4850-8361-f0d2017ea86c">

This warning view should only show if you are burning CANNOT_BURN_FUSES. It is currently always showing on "revoke-pcc" flow